src/content/en/fundamentals/discovery-and-monetization/payment-request/payment-request-ux-considerations.md

```diff
@@ -0,0 +1,514 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2017-06-28 #}
+{# wf_published_on: 2017-06-28 #}
+
+
+# Payment Request UX considerations {: .page-title }
+{% include "web/_shared/contributors/mustafa.html" %}
+
+The Payment Request API was fundamentally built for users, helping
+them get through burdensome checkout flows on mobile efficiently,
+while providing merchants with a higher conversion rate.
+
+In a study conducted by Google, we found that <strong>65.9% of users
+stopped partway through purchase on mobile devices</strong>. Of those
+asked in the study as to why they abandoned the purchase, the most
+common responses ranged from the form being difficult to read,
+difficult to understand or was way too long.
+
+To solve these challenges the Payments Request API creates a simpler
+flow. 
+The API will pre-populate payment details for the user allowing
+users to buy items with a simple tap or click.
+
+One way to think about the Payments Request API is that it is
+essentially a way for a browser to manage a user's payment methods,
+making it easier to make payments on the web.
+
+## The two view states 
+In the Chrome UI, the Payments Request API has
+two states: the <em>'receipt'</em> view and the edit view. The
+<em>'receipt'</em> view (shown at left, below) partially obscuring the
+merchant's site. This allows the user to take a quick glance at their
+payment details. Then there is the edit view (shown at right, below),
+which expands full screen and allows the user to edit each section.
+Once the user taps or clicks a call to action from the
+<em>'receipt'</em> view they will enter the edit mode and will be
+forced to continue there.
+
+<div class="attempt-left">
+  <figure>
+    <img src="images/state01-receipt-view.png" 
+    alt="This is an example of the 'receipt' view">
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="images/state02-edit-view.png" 
+    alt="This is an example of the edit view">
+  </figure>
+</div>
+
+## Basic checkout flow
+
+In this example a user navigates to a site, selects an item and goes
+through the payment request checkout flow. Once a user clicks buy, the
+browser takes over. If the user has previously entered details such as
+their prefered payment method, these will appear as the defaults.
+Otherwise the user can add these. Depending on your particular use
+case you can have defaults auto selected or force the user to fill
+them in.
+
+<video controls autoplay loop muted poster="images/basic-intro-poster.png">
```

Comment:
not according to our articles on the site they done :P 
https://developers.google.com/web/resources/including-videos